salt/modules/boto_elb.py

```diff
@@ -686,3 +694,223 @@ def get_instance_health(name, region=None, key=None, keyid=None, profile=None, i
+            else:
+                existing_policy_name = ''
+            break
+
+    if found_listener:
+        new_policy = False
+        pName = None
+        if not disable_policy:
+            if policy.lower() == 'lbcookiestickinesspolicy':
+                pName = 'LBCookieStickinessPolicy-{0}-{1}'.format(lb.name, hash('{0}-{1}'.format(lb.name, cookie)))
+                if existing_policy_name != pName:
+                    p = lb.create_cookie_stickiness_policy(int(cookie), pName)
+                    new_policy = True
+            elif policy.lower() == 'appcookiestickinesspolicy':
+                pName = 'AppCookieStickinessPolicy-{0}-{1}'.format(lb.name, hash('{0}-{1}'.format(lb.name, cookie)))
+                if existing_policy_name != pName:
+                    p = lb.create_app_cookie_stickiness_policy(cookie, pName)
+                    new_policy = True
+            else:
+                msg = 'Invalid policy type {0} specified.'.format(policy.lower())
+                log.debug(msg)
+                return False
+        else:
+            pName = ''
+        if new_policy or disable_policy:
+            log.debug('Attempting to set listener policy for port {0} to policy {1}'.format(listener, pName))
+            lb.set_policies_of_listener(listener, pName)
+        if (existing_policy_name and new_policy) or disable_policy:
+            log.debug('Attempting to delete existing policy {0}'.format(existing_policy_name[0]))
+            lb.delete_policy(existing_policy_name[0])
+        return True
+    else:
+        log.debug('Listener {0} not found'.format(listener))
+    return False
+
+
+def set_elb_tags(name, tags, region=None, key=None, keyid=None, profile=None):
+    '''
+    Add the tags on an ELB
+
+    name
+        name of the ELB
+
+    tags
+        dict of name/value pair tags
+
+    '''
+
+    if exists(name, region, key, keyid, profile):
+        conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+        ret = _add_tags(conn, name, tags)
+        return ret
+    else:
+        return False
+
+
+def delete_elb_tags(name, tags, region=None, key=None, keyid=None, profile=None):
+    '''
+    Add the tags on an ELB
+
+    name
+        name of the ELB
+
+    tags
+        list of tags to remove
+
+    '''
+    if exists(name, region, key, keyid, profile):
+        conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+        ret = _remove_tags(conn, name, tags)
+        return ret
+    else:
+        return False
+
+
+def _build_tag_param_list(params, tags):
+    '''
+    helper function to build a tag parameter list to send
+    '''
+    keys = sorted(tags.keys())
+    i = 1
+    for key in keys:
+        value = tags[key]
+        params['Tags.member.{0}.Key'.format(i)] = key
+        if value is not None:
+            params['Tags.member.{0}.Value'.format(i)] = value
+        i += 1
+
+
+def _get_all_tags(conn, load_balancer_names=None):
+    """
```

Comment:
Can you convert these triple-quotes (`"""`) to single-quotes (`'''`), please? Our standard is to use single quotes. If you could change that in the functions below, as well, that would be fantastic!